controlplane/kubeadm/controllers/controller_test.go

```diff
@@ -379,7 +380,8 @@ func TestReconcileClusterNoEndpoints(t *testing.T) {
 
 	result, err := r.Reconcile(ctrl.Request{NamespacedName: util.ObjectKey(kcp)})
 	g.Expect(err).NotTo(HaveOccurred())
-	g.Expect(result).To(Equal(ctrl.Result{}))
+	// TODO: this should stop to re-queue as soon as we have a proper remote cluster cache in place.
+	g.Expect(result).To(Equal(ctrl.Result{Requeue: true, RequeueAfter: 20 * time.Second}))
```

Comment:
```suggestion
```

Reply:
It required to change the test... we are re-queueing now in this case